scripts/ecnconfig

```diff
@@ -22,6 +24,27 @@ optional arguments:
   -gdrop --green-drop-prob     set drop probability for packets marked green
   -ydrop --yellow-drop-prob    set drop probability for packets marked yellow
   -rdrop --red-drop-prob       set drop probability for packets marked red
+
+2) show and change ECN on/off status on queues
```

Comment:
>2) [](start = 0, length = 2)

It is weird that one script has 2 usages. How about combine them into 1 usage or separate into 2 scripts?

Reply:
The original idea was 2 scripts. Guohan wants to put them together to avoid confusing the users. Then we have two streams of usage (i.e., two usages) here